yarn.lock

```diff
@@ -20804,6 +20988,26 @@ recharts@^1.8.5:
     recharts-scale "^0.4.2"
     reduce-css-calc "^1.3.0"
 
+recharts@^2.1.5:
```

Comment:
Just a note here. The entire rest of the project uses `^1.8.5`, so this pulls in an extra dependency. It's not a huge deal in a lazy-loading world, but it would be nice to stay on one version if possible. Do you know if you could function fine with that older version and then we bump the entire repo at a later time to the 2.x line? If not, then fine to leave this as is

Reply:
1.8.5 is 2.5 years old. I wonder if we could bump them, I can give it a shot, unsure how to test them though apart from them simply compiling 🤔

Reply:
As long as we don't do the bump as part of this PR. Let's keep things constrained